fire/cli/niv/_ilæg_nye_koter.py

```diff
@@ -122,7 +130,7 @@ def ilæg_nye_koter(projektnavn: str, sagsbehandler: str, **kwargs) -> None:
         "Tekst": sagseventtekst,
         "uuid": sagsevent.id,
     }
-    sagsgang = sagsgang.append(sagsgangslinje, ignore_index=True)
+    sagsgang = frame.append(sagsgang, sagsgangslinje)
```

Comment:
Jeg kan ikke lige på siddende mås se hvor "frame" kommer fra - jeg går ud fra at du har stür på det :-)

Reply:
Nåh - nu fik jeg øje på import'en

Reply:
Ja, jeg tænkte over, hvordan det skulle navngives og startede med fdf (for fire dataframe), men så blev det mere obskurt, hvad modulet handler om, når man bruger det. Egentlig kan modulet bruges andre steder, men at lave en selvstendig pakke var nok lidt uden for scope (tid) her.